BAZProjectC1/Extensions/UIImageViewExtension.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIImageViewExtension.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 17/09/22.
+//
+
+import Foundation
+import UIKit
+extension UIImageView{
+    public func loadImageFromUrl(urlString: String){
```

Comment:
Nitpick: Deja un espacio entre el paréntesis y las llaves.
También puedes usar el placeholder para nombrar tu función:
```suggestion
    public func loadImagel(from urlString: String) {
```